src/Sylius/Bundle/WebBundle/Behat/WebContext.php

```diff
@@ -703,6 +704,27 @@ public function iChangeMyPasswordTo($newPassword)
         $this->pressButton('Save changes');
     }
 
+    /**
+     * @When I remove :name from cart
+     */
+    public function iRemoveFromCart($name)
+    {
+        $name = explode(' ', $name)[0];
+        $trLocator = sprintf('table tbody tr:contains("%s")', $name);
+        $tr = $this->getElementByCss($trLocator);
+        $buttonLocator = sprintf('td a.btn-danger');
+        $button = $tr->find('css', $buttonLocator);
+        $button->click();
```

Comment:
`td a.btn-danger` is not a good locator :smile: We should add a text to this link as it is in the backend with `edit` etc. Then we could reuse `iClickNear` method from WebContext https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ResourceBundle/Behat/WebContext.php#L214.